### DISCUSSION TITLE
Run workflow button on actions tab switches to the mobile app, then back to the browser, but fails to run workflow.

### DISCUSSION THREAD

**Discussion by jessehouwing** [community/community#53630] [Mobile]
When I click the run workflow button in Firefox is Chrome on Android the GitHub mobile app tries to take over, fails to lead do what I want it to do and redirects back to the browser. But it fails to actually run the workflow.

This has essentially broken the workflow_dispatch trigger on mobile.

**Comment by taki-on** [comment]
Thank you for your report! I will pass it to the engineers so that we can have a look at it!

**Reply by jessehouwing** [reply]
> > Thank you for your report! I will pass it to the engineers so that we can have a look at it!
> 
> 

This happens with many more buttons be the way. Many in the settings panel, queue workflow, edit code, add custom secrets patterns, approving organizational pat tokens...

It's bloody irritating.
